catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/service/SearchService.java

```diff
@@ -101,10 +102,8 @@
     /**
      * Creates a new SearchService
      *
-     * @param filterBuilder
-     *            - FilterBuilder to use for queries
-     * @param searchController
-     *            - SearchController to handle async queries
+     * @param filterBuilder    - FilterBuilder to use for queries
```

Comment:
✏️  Since we're touching those comments, can we remove the `-`? They aren't needed.